ext/standard/basic_functions.stub.php

```diff
@@ -60,6 +60,25 @@ function stream_wrapper_restore(string $protocol): bool {}
 /** @return int|false */
 function array_push(array &$stack, ...$args) {}
 
+function krsort(array &$arg, int $sort_flags = PHP_SORT_REGULAR): bool {}
```

Comment:
`PHP_SORT_REGULAR` is a C macro; its PHP pendant is `SORT_REGULAR`. I'm taking the liberty to fix that myself. :)

Reply:
Got it. Thanks.